plugins/VstBase/RemoteVstPlugin.cpp

```diff
@@ -1237,10 +1239,14 @@ void RemoteVstPlugin::getParameterLabels()
 
 void RemoteVstPlugin::sendCurrentProgramName()
 {
-	char presName[64];
-	sprintf( presName, "%d/%d: %s", pluginDispatch( effGetProgram ) + 1, m_plugin->numPrograms, programName() );
-
-	sendMessage( message( IdVstCurrentProgramName ).addString( presName ) );
+	char presName[64] = {0};
```

Comment:
```suggestion
	char presName[64] = {};
```
I know `{0}` is common in C, but it's rarely used in C++.